src/api/app/views/webui/request/_actions_details.html.haml

```diff
@@ -11,15 +11,19 @@
           %h6.dropdown-header
             %span Action
             %span.float-right Revision
-        - submit_actions.each do |action_item|
+        - actions.each do |action_item|
           %li
             %hr.dropdown-divider
-            = link_to((render partial: 'action_text', locals: { action: action_item }),
-                      request_show_path(number: bs_request.number,
-                                        request_action_id: action_item.id,
-                                        full_diff: diff_limit,
-                                        diff_to_superseded: diff_to_superseded_id),
-                      class: "dropdown-item #{action_item.id == active_action.id ? 'disabled' : ''}")
+            - if action_item.type == 'submit'
+              = link_to((render partial: 'action_text', locals: { action: action_item }),
+                        request_show_path(number: bs_request.number,
+                                          request_action_id: action_item.id,
+                                          full_diff: diff_limit,
+                                          diff_to_superseded: diff_to_superseded_id),
+                        class: "dropdown-item #{action_item.id == active_action.id ? 'disabled' : ''}")
+            - else
+              %span.dropdown-item.disabled
+                = render partial: 'action_text', locals: { action: action_item }
```

Comment:
We purposefully skip everything beside `BsRequestActionSubmit` as we have no plan yet how to display them, no need to bring them in for this single thing.